test/common/upstream/outlier_detection_impl_test.cc

```diff
@@ -483,8 +485,10 @@ TEST_F(OutlierDetectorImplTest, ConnectSuccessWithOptionalHTTP_OK) {
  * code.
  */
 TEST_F(OutlierDetectorImplTest, ExternalOriginEventsNonSplit) {
+  ON_CALL(runtime_.snapshot_, getInteger(MaxEjectionPercentRuntime, _)).WillByDefault(Return(60));
   EXPECT_CALL(cluster_.prioritySet(), addMemberUpdateCb(_));
   addHosts({"tcp://127.0.0.1:80"});
+  addHosts({"tcp://127.0.0.2:80"});
```

Comment:
same host as above?

Reply:
should I change the port?

Right now the IPs are different. Would `127.0.0.1:81` be better?